app/models/ability.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    # Define abilities for the user here. For example:
+    #
+    return unless user.present?
+
+     can :read, :all
+     can :create, [Post, Comment]
+
+     if user.role = 'admin'
```

Comment:
- [ ] [ OPTIONAL] You can also create a method in user controller named`admin?` and use it here as  `user.admin?`, This way you can use this method anywhere.
```ruby
def admin?
     role == 'admin'
end
```